src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -17,10 +17,16 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 		return next(new Error(Errors.MISSING_GITHUB_TOKEN));
 	}
 
+	const { issue_key: key, issue_summary: summary } = req.query;
+	if (!key ||!summary) {
```

Comment:
It's not strictly necessary that the issue summary exists, the key is a must, the summary is nice to have. None of this is on the story though lolol.

As a side note, there is also work thats happening in a different team that can manipulate these query params, so it can pass different issue values.

Even if the above work gets done, the only required field that remains is **Issue key**.

```suggestion
	if (!key) {
```